packages/web-api-client/src/a11y-service-credential.ts

```diff
@@ -40,4 +33,28 @@ export class A11yServiceCredential {
 
         return request.defaults(authOptions);
     }
+
+    private async getTokenWithRetries(numRetries: number): Promise<TokenResponse> {
+        try {
+            return await this.tryGetToken();
+        } catch (err) {
```

Comment:
Are we doing retries for all kinds of errors, not only 429 and 5xx?

Reply:
And we should wait for at least 1 sec before retry as suggested here: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#retry-guidance

Reply:
As far as I can tell, there doesn't seem to be an easy way to get the error code from the response. The ErrorResponse object from adal-node only has error and errorDescription (both strings).

Reply:
Error description is created as: 
````
var returnErrorString = operationMessage + ' request returned http error: ' + response.statusCode;
````

Not saying we should parse it, just some thing to consider. I'm fine retry on all errors.